src/plugin/report-plugin/Listener/ReportListener.ts

```diff
@@ -145,6 +145,23 @@ export default class ReportListener {
             return;
         }
 
+        // Only allow Hotline members to react
+        const hotline = this.client.guilds.get('204100839806205953');
+            
+        try {
```

Comment:
Why is this wrapped in a trycatch?

Reply:
That's the way it already was in the command handler check.

Reply:
https://github.com/DiscordHotline/bot/blob/73f9556969ed8fb9f57765a175aff74df7235f60/src/Kernel.ts#L58

Reply:
I presume it's in the event the Hotline guild is null for some reason, maybe because it's unavailable.

Reply:
To get rid of the unneeded try/catch (no idea why it's in that other file, will probably remove it in a cleanup), you can do the following:
```js
const hotline = this.client.guilds.get('204100839806205953');

if (hotline) {
  const member = hotline.members.get(userId);

  if (!member) {
    return;
  }

  // Only allow members who are in the Member role.
  if (!member.roles.includes('531617261077790720')) {
    return;
  }
}
```

It's basically the same logic, but just without the try/catch block around it